app/src/main/kotlin/no/nav/arbeidsgiver/notifikasjon/produsent/api/ProdusentAPI.kt

```diff
@@ -30,6 +30,7 @@ object ProdusentAPI {
         return TypedGraphQL(
             createGraphQL("/produsent.graphql") {
                 directive("Validate", ValidateDirective)
+//                directiveWiring(ValidateDirective) // this eliminates the need for @Validate on every field
```

Comment:
dersom man hadde benyttet `directiveWiring` i stedet for `directive("Validate"` så hadde man nok ikke plutselig manglet validering pga Validate annotasjonen mangler på felt. Men vi skal uansett skrive dette om så kanskje ikke noe vits å stresse med det. Rydder det i neste runde